src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -465,20 +474,48 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
                 diagnostics,
                 useMethodSignatureParameterTypes: false,
                 methodSignatureOpt: null,
-                returnTypes: null,
                 initialState: null,
-                analyzedNullabilityMapOpt: null);
+                analyzedNullabilityMapOpt: null,
+                takeIncrementalSnapshots: false,
+                snapshotManagerOpt: ref ignored,
```

Comment:
>ignored [](start = 40, length = 7)

Should we ```Free``` this?

Reply:
No longer relevant.

---
In reply to: [288785918](https://github.com/dotnet/roslyn/pull/35850#discussion_r288785918) [](ancestors = 288785918)